src/Service/Models/RegisterToken.cs

```diff
@@ -62,13 +62,13 @@ public class VerifySignInToken : Token
 public class Token
 {
     [Key(0)]
-    public required DateTime ExpiresAt { get; set; }
```

Comment:
@jonashendrickx @Tyrrrz What effect will this have?

Reply:
Not entirely sure how it works in the absence of NRTs, but most likely:
- If ASP.NET Core: disables built-in validation that this field must be present in the payload
- If EF Core: disables automatic `NOT NULL` constraint

Reply:
<img width="1779" alt="image" src="https://github.com/passwordless/passwordless-server/assets/1045327/00baf2ee-ecc5-4802-b682-4f12b94a71b8">

But no impact elsewhere, it just fixes this bug, we were not passing values to these 3 fields ever.

Reply:
Alright, we'll look into them.